modules/account-lib/src/coin/near/transaction.ts

```diff
@@ -62,7 +62,7 @@ export class Transaction extends BaseTransaction {
       parsedAction = {
         functionCall: {
           methodName: functionCallObject.methodName,
-          args: JSON.parse(functionCallObject.args.toString()),
+          args: JSON.parse(Buffer.from(functionCallObject.args).toString()),
```

Comment:
Whats this for?

Reply:
Sometimes buffer can come as a buffer or a string buffer, we could try to modify every case where args need to be parsed or just use Buffer.from() here which accept both types